src/pages/iou/IOUDetailsModal.js

```diff
@@ -67,21 +67,33 @@ const propTypes = {
     session: PropTypes.shape({
         /** Currently logged in user email */
         email: PropTypes.string,
-    }).isRequired,
+    }),
 
     /** Actions from the ChatReport */
     reportActions: PropTypes.shape(reportActionPropTypes),
 
     /** Information about the network */
     network: networkPropTypes.isRequired,
 
+    /** chatReport associated with iouReport */
+    chatReport: PropTypes.shape({
+        /** Report ID associated with the transaction */
+        reportID: PropTypes.string,
+    }),
+
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
     iou: {},
     reportActions: {},
     iouReport: undefined,
+    session: {
+        email: null,
+    },
+    chatReport: {
+        participants: [],
```

Comment:
Why is the default value only `participants` and the type for `chatReport` is only reportID?

Reply:
Okay, let's just fix what the lint complains about.

Reply:
It's not clear either what the function needs from the report data. I'll take a look deeper into this.

Reply:
@Prince-Mendiratta, we need all the keys from `chatReport`. For example, `getPayMoneyRequestParams` is updating the report data.

Reply:
@grgia Any suggestion what we should put for `chatReport` type?

Reply:
@Prince-Mendiratta At this moment, we can use the above snippet code.